test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/BatchTests.cs

```diff
@@ -42,5 +88,32 @@ public async Task WhenSubmittingABatch_GivenANonBundleResource_ThenBadRequestIsR
 
             Assert.Equal(HttpStatusCode.BadRequest, ex.StatusCode);
         }
+
+        private void ValidateResourceOutput(Bundle resource)
```

Comment:
nit: is there advantage of splitting this into a separate method? It's only used by `WhenSubmittingABatch_GivenAValidBundle_ThenSuccessIsReturnedForBatchAndExpectedStatusCodesPerRequests` and specifically validating the result of that test so seems like the logic just belongs to that method?